src/components/Item/VisualizationItem/Item.js

```diff
@@ -328,6 +337,7 @@ Item.contextTypes = {
 
 Item.propTypes = {
     activeType: PropTypes.string,
+    countInDashboard: PropTypes.bool,
```

Comment:
What about naming this "gatherDataStatistics"?

Reply:
Sounds good. Renamed.